pkgs/top-level/aliases.nix

```diff
@@ -768,9 +768,12 @@ mapAliases {
   MIDIVisualizer = midivisualizer; # Added 2024-06-12
   mikutter = throw "'mikutter' has been removed because the package was broken and had no maintainers"; # Added 2024-10-01
   mime-types = mailcap; # Added 2022-01-21
-  minetest-touch = minetestclient; # Added 2024-08-12
-  minetestclient_5 = minetestclient; # Added 2023-12-11
-  minetestserver_5 = minetestserver; # Added 2023-12-11
+  minetest = luanti; # Added 2024-11-11
+  minetestclient = luanticlient; # Added 2024-11-11
+  minetestserver = luantiserver; # Added 2024-11-11
+  minetest-touch = luanticlient; # Added 2024-08-12
+  minetestclient_5 = luanticlient; # Added 2023-12-11
+  minetestserver_5 = luantiserver; # Added 2023-12-11
```

Comment:
I think these should be replaced with throws. Take a look at `mod_dnssd` which is a few lines down for an example.

Reply:
Thanks. I'm tweaking them to throws and fixing the formatting complaint

Any thoughts from people on dropping the `_5` items completely vs adding a `luanticlient_5` alias?
Also what to do with `minetest-touch`.
And maybe while we're renaming stuff, `minetestclient` -> `luanti-client` would be nice IMO

cc @fpletz @fgaz

Reply:
> Any thoughts from people on dropping the `_5` items completely vs adding a `luanticlient_5` alias? Also what to do with minetest-touch.

I'd drop both

> And maybe while we're renaming stuff, minetestclient -> luanti-client would be nice IMO

On one hand the server executable is named minetestserver so it makes sense to follow the same pattern. On the other, [all other distro packages](https://repology.org/project/luanti/versions) are dashed.

I agree with renaming.

Reply:
I dropped the _5 aliases completely, but I did not make the others throw since the module still uses `pkgs.minetest`